lib/rihanna/migration.ex

```diff
@@ -104,6 +104,7 @@ defmodule Rihanna.Migration do
       CREATE TABLE #{table_name} (
         id int NOT NULL,
         term bytea NOT NULL,
+        priority integer NOT NULL DEFAULT 19,
```

Comment:
Why is the default 19?

Reply:
To conform with the scale used in `nice`: https://linux.die.net/man/1/nice

Could be any other value.